### PR TITLE
Align distribution build structure.

### PR DESCRIPTION
**Pull request by LarsSkovslund**
This PR implements #44 with the following changes;  
- `Images` folder added
- Nullable set to `annotations` for test projects.
- Added support for nuget package configuration

### CODE REVIEW

File: distribution/Directory.Build.props [hunk 1]
```diff
@@ -35,12 +40,12 @@
   <!-- Shared code analyzers used for all projects in the solution -->
   <ItemGroup Label="Code Analyzers">
     <PackageReference Include="AsyncFixer" Version="1.5.1" PrivateAssets="All" />
-    <PackageReference Include="Asyncify" Version="0.9.7" PrivateAssets="All" />
-    <PackageReference Include="Meziantou.Analyzer" Version="1.0.606" PrivateAssets="All" />
+    <PackageReference Include="Asyncify" Version="0.9.7" PrivateAssets="all" />
+    <PackageReference Include="Meziantou.Analyzer" Version="1.0.606" PrivateAssets="all" />
```
Comment (davidkallesen) [inline distribution/Directory.Build.props:44]: Is there a reason for lower "all" in some special cases?

File: distribution/src/Directory.Build.props [hunk 2]
```diff
@@ -6,20 +6,67 @@
...
+  <ItemGroup>
+    <!-- Ensures package icon is valid -->
+    <None Include="..\..\images\icon.png" Pack="true" PackagePath="\" Visible="false" />
+  </ItemGroup>
+
+  <PropertyGroup Label="Build instructions">
+    <OutputType>Library</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <!-- creates a regular package and a symbols package -->
```
Comment (davidkallesen) [inline distribution/src/Directory.Build.props:39]: creates -> Creates

File: distribution/Directory.Build.props [hunk 3]
```diff
@@ -1,10 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <PropertyGroup Label="Metadata configuration">
+    <OrganizationName>atc-net</OrganizationName>
+    <RepositoryName>atc-cosmos</RepositoryName>
```
Comment (davidkallesen) [inline distribution/Directory.Build.props]: Don't think this should be default values - suggestion:

- <OrganizationName>[INSERT-YOUR-VALUE]</OrganizationName>
- <RepositoryName>[INSERT-YOUR-VALUE]</RepositoryName>

File: distribution/src/Directory.Build.props [hunk 4]
```diff
@@ -6,20 +6,67 @@
...
     <IncludeSymbols>true</IncludeSymbols>
+    <!-- Creates symbol package in the new .snupkg format -->
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <!--
+      instruct the build system to embed project source files that are not tracked by the source control
```
Comment (davidkallesen) [inline distribution/src/Directory.Build.props:44]: instruct -> Instruct

File: distribution/Directory.Build.props (hunk 1 above)
Comment (LarsSkovslund) [reply distribution/Directory.Build.props:44]: No - it's just not relevant as long as it is spelled correctly.

File: distribution/src/Directory.Build.props (hunk 2 above)
Comment (egil) [reply distribution/src/Directory.Build.props]: David, would be a great help if you used the "suggestion" features for these kinds of things.

**Review by egil — APPROVED** [review]

**Review by davidkallesen — APPROVED** [review]

**Pull request merged**